src/assignments/launch-checklist.rst

```diff
@@ -0,0 +1,118 @@
+Assignment #5: Launch Checklist Form
```

Comment:
Sally, the assignment looks good.  I found a few more minor wording issues that I missed in the last review, but they are really quick fixes.

Approval granted!